### ISSUE TITLE
make sure the new default config remains overridable in .config for all images

### ISSUE THREAD

**Issue by RoyalOughtness** [wayblueorg/wayblue#20]

**Comment by rompetomp** [comment]
How are we supposed to provide a config for hyprland before this issue is fixed? Or is it just not possible yet?

**Comment by RoyalOughtness** [comment]
> How are we supposed to provide a config for hyprland before this issue is fixed? Or is it just not possible yet?

Not possible yet but I can get around to it soon (next few days). PRs are also welcome. It's a very straightforward fix. All we need is for the SDDM entry to invoke a script that checks for the presence of a config in .config, then falls back to the current config from /usr if there's no config there.

Also, these images are all very much in an alpha state and I just realized that isn't made clear in the readme. Added that here: https://github.com/wayblueorg/wayblue/blob/live/README.md
